app/assets/javascripts/discourse/app/templates/components/sidebar/footer.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar-footer-wrapper">
+  <div class="sidebar-footer-container">
+    <div class="sidebar-footer-links">
+      <LinkTo @route="about" title={{i18n "about.simple_title"}} class="sidebar-footer-link sidebar-footer-link-about">
+        {{i18n "about.simple_title"}}
+      </LinkTo>
+
+      <a href
+        {{action (route-action "showKeyboardShortcutsHelp")}}
+        title={{i18n "keyboard_shortcuts_help.title"}}
+        class="sidebar-footer-link sidebar-footer-link-keyboard-shortcuts" >
```

Comment:
This is tricky... I think a button makes more sense for this, but we want this button to look like a regular link which I don't think is possible with `<DButton/>` 🤔 

Maybe it makes sense to swap this with the settings link below? That way, the settings link becomes a proper link and this becomes a proper button, but this is definitely a design decision. I think we can revisit this later.